<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>سیستم مدیریت بیمارستان</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>سیستم مدیریت بیمارستان</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px; font-weight: bold; color: #2c3e50; margin: 20px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="nav_layout">
      <item>
       <widget class="QPushButton" name="btn_patients">
        <property name="text">
         <string>مدیریت بیماران</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #3498db; color: white; padding: 15px; font-size: 14px; border-radius: 5px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_doctors">
        <property name="text">
         <string>مدیریت پزشکان</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #2ecc71; color: white; padding: 15px; font-size: 14px; border-radius: 5px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_appointments">
        <property name="text">
         <string>مدیریت نوبت‌ها</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #e74c3c; color: white; padding: 15px; font-size: 14px; border-radius: 5px; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="search_layout">
      <item>
       <widget class="QLineEdit" name="search_input">
        <property name="placeholderText">
         <string>جستجو...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>جستجو</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #95a5a6; color: white; padding: 8px; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="operation_layout">
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>افزودن</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #27ae60; color: white; padding: 10px; font-size: 12px; border-radius: 3px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_edit">
        <property name="text">
         <string>ویرایش</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #f39c12; color: white; padding: 10px; font-size: 12px; border-radius: 3px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>حذف</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: #e74c3c; color: white; padding: 10px; font-size: 12px; border-radius: 3px; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>